CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  - Adding shortcut for loading Sprites and SpriteAnimation from the global cache
  - Adding loading methods for the different `ParallaxComponent` parts and refactor how the delta velocity works
  - Add tests for `Timer` and fix a bug where `progress` was not reported correctly
+ - Removing methods initialDimensions and removeGestureRecognizer to avoid confusion
```

Comment:
I would personally add the method names in back ticks

Reply:
Done